crates/astria-composer/src/executor/bundle_factory/mod.rs

```diff
@@ -24,17 +33,34 @@ enum SizedBundleError {
     SequenceActionTooLarge(SequenceAction),
 }
 
+pub(super) struct RollupCountsReport(HashMap<RollupId, u32>);
```

Comment:
Can this remain an implementation deatil of this module? Can you make it non-`pub(super)`?